design/decisions/why-license/index.qmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "Why MIT License"
+description: |
+  Open and permissive licensing of copyrighted content is a way to increase use of a software product, ease collaboration and re-use, and share intellectual ideas with others. The MIT License is a very permissive copyright license.
+author: "Kristiane Beicher"
+date: "2023-04-01"
+date-modified: last-modified
+categories:
+  - licensing
+  - copyright
+  - software-architecture
+---
+
+## Context and Problem Statement
+
+When developing a new piece of software it is important to consider the following question as early as possible.
+
+> What kind of re-use will we ultimately allow other developers to make of our code/product?
+
+In order to control this use, it is important to have considered which license type we want our product to be available under, as this will tell the rest of the development community what they can (and can't) do with our code. 
+
+## Decision Drivers
+
+Without a license, other users will not be allowed to (easily) (re-)use, modify, contribute, or enhance our software due to copyright reasons. Our mission isn't only to build a product; we also aim to develop a creative community around Seedcase and the topics we work on. We want people to be able to freely and smoothly contribute enhancements and other improvements that can be incorporated into future releases of Seedcase.  It is also our intention to make the Seedcase software available to commercial enterprises, which means that we need to carefully consider which license we adopt.
+
+Below is a walk-through of the different types of licenses that we believe we will need and a discussion on the reasons we make the choices
+that we do.
+
+## Considered Options
+
+### Types of licenses
+
+The Open Source Initiative [approves](https://opensource.org/licenses) a specific set of licenses that determine whether a project can be called "open source". These licenses are our starting point on deciding which license to use and how permissive we want it to be, in terms of who can work on our code and how it can be used.
```

Comment:
Add abbreviation of Open Source Initiative --> "OSI" and then use throughout? :)